crates/sui-node/src/lib.rs

```diff
@@ -1115,6 +1119,44 @@ impl SuiNode {
         assert_eq!(next_epoch, new_epoch_store.epoch());
         new_epoch_store
     }
+
+    fn check_system_consistency(&self) {
+        if !self.config.enable_expensive_safety_checks && cfg!(not(debug_assertions)) {
```

Comment:
Will this run on the `stress` + `simtest` nightly (which I think runs in debug mode and might get rather expensive)? Not necessarily suggesting we should turn it off, but worth understanding.

Reply:
Yes. It will, assuming debug_assertions is enabled in that